### ISSUE TITLE
Schema properties don't indicate if property can be null

### ISSUE THREAD

**Issue by colin-mckibben-sp** [PaloAltoNetworks/docusaurus-openapi-docs#387]
### Is your feature request related to a problem?

OpenAPI supports the ability to mark schema properties as null by setting the `nullable: true` property.  We used to use Redoc to generate our spec, which indicated which properties can be null with a tag:

![image](https://user-images.githubusercontent.com/75683148/213563465-5bbbaead-fb8b-4ab4-9274-c0b8502df7eb.png)

Docusaurus has no visual indication if a property can be null.

![image](https://user-images.githubusercontent.com/75683148/213563765-d4ec3986-7308-4789-b7b5-8f47840243c1.png)

The OpenAPI definition for the `scope` property is as follows:

```yaml
scope:
    type: array
    nullable: true
    items:
      type: string
      default: "sp:scopes:all"
    description: Scopes of the personal  access token.
    example: [ "demo:personal-access-token-scope:first", "demo:personal-access-token-scope:second" ]
```

### Describe the solution you'd like

I would like to see a visual indicator for each property that can be null.

### Describe alternatives you've considered

None

### Additional context

Knowing which properties can potentially return `null` is helpful for consumers of the spec so they know when they need to handle these edge cases in their code.

**Comment by sserrata** [comment]
Hi @colin-mckibben-sp, thanks for reporting this issue. I put together a quick poc to collect some feedback. Was also hoping you could help clarify a few things:

* Should `nullable` be applicable to both request and response schema items?
* Can `object` data types be `nullable`?
* Does the UI/UX in the screencap below work?

https://user-images.githubusercontent.com/9343811/213803671-cfa899ff-4136-4016-9ca3-ad02d59c9ba5.mov

**Comment by colin-mckibben-sp** [comment]
Hi @sserrata,

- Should nullable be applicable to both request and response schema items?
  - Yes, I think showing nullable on both request and response is appropriate.  I don't think OpenAPI 3 limits nullable to either one, it's just a schema level property.
- Can object data types be nullable?
  - Yes, at least our API can return null for objects.  Again, I don't see the OA3 spec saying it can't be used for objects.  Looks like it works for all JSON types, including array.
- Does the UI/UX in the screencap below work?
  - Yes, that works for me.  It's close to how Redoc shows it, so I think that will make it easier to understand for users who are also comfortable with redoc generated sites.

Thank you for picking this up so quickly!

**Comment by sserrata** [comment]
Available now in canary release `0.0.0-542` - will include in next full release
